airtable/sync_galaxy_digital_data.js

```diff
@@ -0,0 +1,149 @@
+// Script used to convert Volunteer data from the 
+// (Galaxy Digital)[http://api2.galaxydigital.com/volunteer/docs/] APIs
+// into the AirTable schemas
+import { globalConfig } from '../sync_data/frontend/node_modules/@airtable/blocks';
+import { Airtable } from '@airtable';
+
+const AIRTABLE_API_KEY = globalConfig.get("AIRTABLE_API");
+const AIRTABLE_BASE_ID = globalConfig.get("AIRTABLE_BASE");
+const GALAXY_DIGITAL_API_KEY= globalConfig.get("GALAXY_DIGITAL_API_KEY");
+
+const DESTINATION_TABLE = 'Volunteer Engagement';
+const AIRTABLE_MAX = 10;
+const GALAXY_DIGITAL_MAX = 50;
+
+const base = new Airtable({apiKey: AIRTABLE_API_KEY}).base(AIRTABLE_BASE_ID);
+
+async function getGalaxyDigitalVolunteerData(limit, offset) {
+  const url = new URL("volunteer/user/list/", "https://api2.galaxydigital.com")
+  const params = url.searchParams;
+  params.set('key', GALAXY_DIGITAL_API_KEY);
+  params.set('limit', limit);
+  params.set('offset', offset);
+
+  // TODO report errors
+  const response = await fetch(url, {method: 'GET'}).then(resp => resp.json()).catch(return null, error);
+  return [response, null];
```

Comment:
It's been a while since I've written pure javascript, is this an anti-pattern returning a tuple here?  I've been writing a lot of Go lately so the style of this script mirrors that